src/mame/sgi/indy_indigo2.cpp

```diff
@@ -156,6 +156,7 @@ class ip24_state : public driver_device
 
 	uint8_t m_volume_l = 0;
 	uint8_t m_volume_r = 0;
+	unsigned SYSTEM_CLOCK = 50'000'000;
```

Comment:
I don’t think having this here is a great idea:
* You’ve formatted it like a constant when it isn’t.
* It’s never used outside machine configuration functions.

You’re better off just using local `constexpr` variables in the functions themselves.

Reply:
Actually it's used in memory controller's initialization routine.

Reply:
I see, passing clock frequency as a parameter to `ip24_state::ip24_base` makes more sense.

Reply:
No, this variable is not used by the memory controller.  You’re passing its value to memory controller member functions, but the variable itself is only used in machine configuration functions in this class.  It’s being done in a confusing way.